docs/install/usb-install.md

```diff
@@ -70,3 +74,26 @@ If a USB flash driver boots, but you can't see the harvester installer, try one
 
 - Plug the USB flash drive into a USB 2.0 slot.
 - For version `v0.3.0` or above, remove the `console=ttyS0` parameter when booting. Press `e` to edit the GRUB menu entry and remove the `console=ttyS0` parameter.
+
+### Harvester interactive ISO hangs with the USB installation method
+
+During installation from a USB flash drive with v1.2.0 ISO image (created by tools like `balenaEtcher`, `dd`, etc.), the installation process may get stuck on the initial image loading process. This happens because a required label is missing on the boot partition. Therefore, the installation cannot mount the data partition correctly, causing some checks in "dracut" to be blocked.
+
+If you encounter this issue, you'll observe a screen similar to the screenshot below, and the process will hang for at least 50 minutes (the default timeout value from `dracut`).
```

Comment:
```suggestion
During installation from a USB flash drive with v1.2.0 ISO image (created by tools like `balenaEtcher`, `dd`, etc.), the installation process may get stuck on the initial image loading process because a required label is missing on the boot partition. Therefore, the installation cannot mount the data partition correctly, causing some checks in `dracut` to be blocked.

If you encounter this issue, you'll observe the following similar output, and the process will hang for at least 50 minutes (the default timeout value from `dracut`).
```